src/current/cockroachcloud/egress-private-endpoints.md

```diff
@@ -131,6 +131,28 @@ curl https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/networking/egress-
 }'
 ~~~
 
+#### Amazon CloudWatch logs export endpoint
+
+Log export to Amazon CloudWatch requires one private service endpoint per CockroachDB Cloud region, with each endpoint mapping to a respective CloudWatch instance for that region. Since CloudWatch is an AWS-managed service, they are scoped to the AWS account where the endpoint is created. The access keys on the export dictate which CloudWatch account receives the logs.
+
+To export all logs across multiple regions to a single CloudWatch instance, [configure custom DNS](#configure-custom-dns) that maps a CloudWatch instance's domain name for each region to the single instance. In this situation, the `logexport` endpoint sets the `region` field to the region of the CloudWatch instance. 
```

Comment:
I don't see the `logexport` endpoint referenced anywhere on this page. Are you able to link this to an API reference? Also, it's not clear (to me) what the below snippet is showing. Suggest introducing the snippet e.g. "The following snippet exports all logs to a CloudWatch instance in `us-east-1`". Not sure if that's correct.

```
In this situation, the `logexport` endpoint sets the `region` field to the region of the CloudWatch instance. 
```

Reply:
Added a link